client/stylesheets/_toCat.scss

```diff
@@ -5724,6 +5730,10 @@ textarea {
     color: #666 !important;
 }
 
+.headerNav > ul > li > a, .headerNav > ul > li > a:visited, .headerNav > ul > li > a:link {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Selector `headerNav` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line